packages/overlay/src/integrations/sentry/components/insights/webVitals/index.tsx

```diff
@@ -55,6 +55,19 @@ const WebVitals = () => {
     );
   }, [events, sort]);
 
+  const handleRowClick = (measurementEvent: SentryEventWithPerformanceData) => {
+    navigate(`/insights/webvitals/${measurementEvent.event_id}`);
+  };
+
+  const handleRowKeyDown = (
+    e: KeyboardEvent<HTMLTableRowElement>,
+    measurementEvent: SentryEventWithPerformanceData,
+  ) => {
+    if (e.key === 'Enter') {
+      handleRowClick(measurementEvent);
+    }
+  };
+
```

Comment:
Could this also be extracted into a separate component as we seem to be repeating it?